app/services/witter.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios').default;
+const logger = require('../logger');
+
+const WitterInstance = axios.create({
+  baseURL: 'http://numbersapi.com'
+});
+
+exports.getWitter = (num = Math.floor(Math.random() * 100)) =>
+  WitterInstance.get(`/${num}`)
+    .then(res => res.data)
+    .catch(error => {
+      logger.error(`Request Error => ${error.message}`);
+      throw new Error(`Error in getWitter method => ${error.message}`);
```

Comment:
Well done for catching the error! Maybe you can create a custom error in the file: `app/errors.js`, with an internal code we can know that the error is from the number api and not from the server